sample/rust/wca_stdx_trivial_sample/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use wca::prelude::*;
+
+fn echo( () : (), args : Args, _ : Props ) -> Result< (), () > 
+{
+  let mut args = args.0.into_iter();
+  wca::parse_args!( args, value: String );
+
+  println!( "{value}" );
+
+  Ok( () )
+}
+
+fn main() 
+{
+  let args = std::env::args().skip( 1 ).collect::< Vec< _ > >().join( " " );
+
+  let aggregator = wca::stdx::cli( () ).command( echo.arg( "string", Type::String) ).build();
```

Comment:
Please, add detailed documentation for related code. I mean `command`, `arg` .... The user should know how to use the API. 
Please, add some tests for the builder.